examples/next/pages/ui/components/authenticator/custom-sign-up-fields/index.page.tsx

```diff
@@ -28,7 +28,7 @@ export default function App() {
 
                 {/* Append & require Terms & Conditions field to sign up  */}
                 <CheckboxField
-                  errorMessage={validationErrors.acknowledgement}
+                  errorMessage={validationErrors.acknowledgement as string}
```

Comment:
nit: Do we need it? I see it's already recognized as `string` on my local

<img width="742" alt="Screen Shot 2022-03-09 at 15 59 40" src="https://user-images.githubusercontent.com/11983489/157559982-f014c7bb-e4f7-4fc9-b8d8-f8e763f35ec4.png">

Reply:
It won't build. So I had to add this.